rskj-core/src/test/java/co/rsk/peg/PegUtilsLegacyTest.java

```diff
@@ -1793,14 +1808,72 @@ void testIsMigrationTx() {
     }
 
     @Test
-    void testIsPegOutTx() {
+    void isMigrationTx_fromP2shErpFederation_toP2shP2wshFederation() {
+        Context btcContext = new Context(networkParametersMainnet);
+        FederationContext federationContext = FederationContext.builder()
+            .withActiveFederation(p2shP2wshErpFederation)
+            .withRetiringFederation(p2shErpFederation)
+            .build();
+
+        BtcTransaction migrationTx = migrationTransactionBuilder
+            .withActiveFederation(federationContext.getActiveFederation())
+            .withRetiringFederation(federationContext.getRetiringFederation().get())
+            .build();
+
+        Wallet federationsWallet = new BridgeBtcWallet(btcContext, federationContext.getLiveFederations());
+
+        assertTrue(isMigrationTx(
+            migrationTx,
+            federationContext,
+            federationsWallet,
+            bridgeConstantsMainnet.getMinimumPeginTxValue(allActivations),
+            allActivations
+        ));
+    }
+
+    @Test
+    void isMigrationTx_fromP2shP2wshFederation_toP2shP2wshFederation() {
```

Comment:
Sorry, I mean this one. Got confused when moved from the IDE to github to let the comment